src/js/index.js

```diff
@@ -491,47 +538,52 @@ function systemImportHandler(ev)
 
 function importFile(file, type)
 {
-    if(storeCompatibleFile(file)){        
-        if(type == "Project"){
-            // Show the imported file on Dropdown button
-            document.querySelector(".dropzone").classList = "dropzone";
-            document.querySelector(".dropzone p").innerHTML="Drag Audio File Here";
-            document.querySelector("#file-dropdown .dropdown-current span").style = "color: " + theme.getPropertyValue("--text-color");
+    var newFile = ffmpegToWAV(file);
 
-        }
-        else {
-            // Show the imported file on the dragzone
-            document.querySelector(".dropzone").classList = "dropzone full";
-            document.querySelector(".dropzone p").innerHTML = window.audioFile.fileName;
+    if(!newFile || !storeOutputIfExists(newFile)){      
+        raiseAlert("Error!", "Error while processing Input file!");
+        return;
+    }  
+
+    if(type == "Project"){
+        // Show the imported file on Dropdown button
+        document.querySelector(".dropzone").classList = "dropzone";
+        document.querySelector(".dropzone p").innerHTML="Drag Audio File Here";
+        document.querySelector("#file-dropdown .dropdown-current span").style = "color: " + theme.getPropertyValue("--text-color");
+    }
+    else {
+        // Show the imported file on the dragzone
+        document.querySelector(".dropzone").classList = "dropzone full";
+        document.querySelector(".dropzone p").innerHTML = window.audioFile.originalName;
 
-            var selected = document.querySelector("#file-dropdown .selected");
+        var selected = document.querySelector("#file-dropdown .selected");
 
-            if (selected){
-                selected.classList.remove("selected");
-            }
-            
-            document.querySelector("#file-dropdown .dropdown-current span").innerHTML = "Select from project files";
-            document.querySelector("#file-dropdown .dropdown-current span").style = "color: " + theme.getPropertyValue('--ui-dim-color');
+        if (selected){
+            selected.classList.remove("selected");
         }
-        // Update UI functionality panel
-        var createMarkersButton = document.getElementById("create-markers");
-        createMarkersButton.disabled = true;
-        var markerNumberInput = document.getElementById("marker-number");
-        markerNumberInput.disabled = true;
         
-        markerNumberInput.classList.add("active")
+        document.querySelector("#file-dropdown .dropdown-current span").innerHTML = "Select from project files";
```

Comment:
Doesn’t matter for BeatMarker at all, but good to keep in mind for the future. ‘querySelector’ can be pretty slow.

Reply:
Noted.